src/main/java/com/griddynamics/cloud/learning/web/ArticleController.java

```diff
@@ -0,0 +1,32 @@
+package com.griddynamics.cloud.learning.web;
+
+import com.griddynamics.cloud.learning.service.ArticleService;
+import com.griddynamics.cloud.learning.web.dto.ArticleDto;
+import com.griddynamics.cloud.learning.web.dto.ArticleFilterRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@Validated
+@RestController
+@Slf4j
+public class ArticleController {
+
+    private ArticleService service;
+
+    public ArticleController(ArticleService service) {
+        this.service = service;
+    }
+
+    @GetMapping(value = "/articles")
+    public Page<ArticleDto> getArticles(@Valid ArticleFilterRequest request, Pageable pageable) {
+        Page<ArticleDto> articles = service.findArticles(request, pageable);
+        log.info(request.toString());
```

Comment:
remove this line

Reply:
Let's use Sleuth with Zipkin for requests tracing.
Get hands dirty with it.

Tip: You have to add just one dependency to your pom and run a zipkin instance locally
https://zipkin.io/pages/quickstart